src/pages/MyCakesList.js

```diff
@@ -0,0 +1,52 @@
+import "./MyCakesList.css";
+import { Link } from "react-router-dom";
+import { RiFilePaper2Line } from "react-icons/ri";
+import ChCakeImg from "../images/chocolate.png";
+
+export default function MyCakesList() {
+  const recipes = JSON.parse(localStorage.getItem("recipesArray"));
+
+  function renderCakes() {
+    return recipes.map((cake, index) => {
+      const id = index + 1;
+      return (
+        <div className="ItemCard box">
+          <li className="ItemCard" key={cake.id}>
+            <img
+              src={ChCakeImg}
+              alt="description"
+              className="ItemCard__image"
+            />
+
+            <h2 className="ItemCard__name">{cake.cakeName}</h2>
+            <div className="ItemCard--btnDiv">
+              <button className="ItemCard__button delete">X</button>
+
+              <Link to={`/cakes/${id}`}>
+                <RiFilePaper2Line
+                  size="1.6rem"
+                  color="#d84064"
```

Comment:
Maybe add a className to your icon and define your css there?

Reply:
For some reason the styling is not updated when styling with a className. I remember you had a similar issue. Will check how you did it.

Reply:
For size you would need to use font-size.